winui/Avatar-view/customization.md

```diff
@@ -1,10 +1,176 @@
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderThickness](avatarview_images/winui_avatarview_borderthickness.png)
+
+## Background
+
+You can customize the background of the AvatarView control by using the `Background` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         InitialsType="DoubleCharacter"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+          
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom background](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+## Gradient background
+
+You can use the `LinearGradientBrush` or `RadialGradientBrush` for the Background property to specify a range of colors to apply the gradient background for the AvatarView, as shown in the following code example.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView  ContentType="Initials" 
+                          AvatarName="Alex"
+                          AvatarSize="ExtraLarge"
+                          InitialsType="DoubleCharacter">
+            <syncfusion:SfAvatarView.Background>
+                <LinearGradientBrush StartPoint="0,0"
+                                     EndPoint="1,0">
+                    <GradientStop Color="#2F9BDF" Offset="0"/>
+                    <GradientStop Color="#51F1F2" Offset="1"/>
+                </LinearGradientBrush>
+            </syncfusion:SfAvatarView.Background>
+</syncfusion:SfAvatarView>
+  
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarShape = AvatarShape.Circle;
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.AvatarName = "Alex";
+avatarView.Background = new LinearGradientBrush()
+{
+    StartPoint = new Point(0, 0),
+    EndPoint = new Point(1, 0),
+    GradientStops = new GradientStopCollection()
+    {
+        new GradientStop() { Color =  Color.FromArgb(255, 47, 155, 223), Offset = 0 },
+        new GradientStop() { Color = Color.FromArgb(255, 81, 241, 242), Offset = 1 }
+    }
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with Gradient Background](avatarview_images/winui_avatarview_gradient.png)
+
+## Font
+
+You can customize the font of initials in the `AvatarView` control by using the `FontFamily` and `Foreground` properties.
```

Comment:
Customize the font of initials in the `AvatarView` control by using the `FontFamily` and `Foreground` properties.